src/network/pages/SponsorshipsPage.tsx

```diff
@@ -230,6 +230,14 @@ export const SponsorshipsPage = () => {
                         }
                     />
                 </SponsorshipsTableWrap>
+                {sponsorshipsQuery.hasNextPage && (
+                    <LoadMoreButton
+                        onClick={() => sponsorshipsQuery.fetchNextPage()}
```

Comment:
`sponsorshipsQuery.fetchNextPage()` returns a promise. What do you think about try-catching it and if it throws showing an error toast?